apis/conversion/src/index.js

```diff
@@ -23,7 +23,10 @@ const getDefaultExportPropertiesFn = (path) => {
   return undefined; // TODO: add listbox and other
 };
 
-const getExportPropertiesFnc = (qae) => {
+const getExportPropertiesFnc = (qae, convertToTable) => {
+  if (convertToTable && typeof qae.exportTableProperties === 'function') {
+    return qae.exportTableProperties;
```

Comment:
This needs to be exported by the individual charts. For histogram and boxplot this could look like this:
https://github.com/qlik-oss/dist-flow/pull/319